### PR TITLE
Readability improvements - embedded-media-providers

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
- [x] write shorter sentences
- [x]  HTML instead of html
- [x] URL instead of url

Still some issues in this article, but I think we can ignore those 😄 
image

### CODE REVIEW

File: 10/umbraco-cms/extending/embedded-media-providers.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ description: A guide to creating a custom embed providers in Umbraco
 
 # Embedded Media Providers
 
-The Rich Text Editor in Umbraco has an 'Embed' button, that when pressed, slides open a panel to enable editors to paste the Url of a third-party media resource to embed in content.
+The Rich Text Editor in Umbraco has an 'Embed' button, that when pressed, slides open a panel. In this panel enables editors to paste the Url of a third-party media resource to embed in content.
```
Comment (sofietoft) [inline 10/umbraco-cms/extending/embedded-media-providers.md:10]:
```suggestion
The Rich Text Editor in Umbraco has an 'Embed' button, that when pressed, slides open a panel. This panel enables editors to paste the URL of a third-party media resource to embed in content.
```

**Review by sofietoft — COMMENTED** [review]
Looks great @erikjanwestendorp ! 💪 
Just a single suggestion 😄

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for your review. Just updated your the PR with your suggestion 😄

**Comment by sofietoft** [comment]
Thanks @erikjanwestendorp !

Merged it in 😄 👏

**Pull request merged**